### DISCUSSION TITLE
QInput / QSelect icon property

### DISCUSSION THREAD

**Discussion by robsontenorio** [quasarframework/quasar#11885] [Ideas / Proposals]
It would be nice if QInput/QSelect had `icon` property, just like `QBtn`.

I would avoid extra boilerplate using `<template #append>` just for a simple icon.

**Comment by pdanpdan** [comment]
An icon property would need 4 properties
Then 4 more for a label
Then you need to put a tooltip or a button, so you would reduce the boilerplate to only o few dozens properties that would not cover what you need
